data/crac-creation/crac-creator-csa-profiles/src/main/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/CsaProfileConstants.java

```diff
@@ -69,4 +71,102 @@ private CsaProfileConstants() {
 
+    /**
+     * requests for flow cnec
+     */
+
+    public static final String REQUEST_ASSESSED_ELEMENT = "assessedElement";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_WITH_CONTINGENCY = "assessedElementWithContingency";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_IN_BASE_CASE = "inBaseCase";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_NAME = "name";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_OPERATOR = "assessedSystemOperator";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_OPERATIONAL_LIMIT = "operationalLimit";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_IS_CRITICAL = "isCritical";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_NORMAL_ENABLED = "normalEnabled";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_IS_COMBINABLE_WITH_CONTINGENCY = "isCombinableWithContingency";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_WITH_CONTINGENCY_COMBINATION_CONSTRAINT_KIND = "combinationConstraintKind";
+
+    public static final String REQUEST_ASSESSED_ELEMENT_WITH_CONTINGENCY_NORMAL_ENABLED = "normalEnabled";
+
+    public static final String REQUEST_CURRENT_LIMIT = "currentLimit";
+
+    public static final String REQUEST_CURRENT_LIMIT_NORMAL_VALUE = "normalValue";
+
+    public static final String REQUEST_CURRENT_LIMIT_OPERATIONAL_LIMIT_SET = "operationalLimitSet";
+
+    public static final String REQUEST_CURRENT_LIMIT_OPERATIONAL_LIMIT_TYPE = "operationalLimitType";
+
+    public static final String REQUEST_CURRENT_LIMIT_OPERATIONAL_LIMIT_TERMINAL = "terminal";
+
+    public static final String REQUEST_CURRENT_LIMIT_OPERATIONAL_LIMIT_KIND = "kind";
+
+    public static final String REQUEST_CURRENT_LIMIT_OPERATIONAL_LIMIT_DIRECTION = "direction";
+
+    public static final String REQUEST_CURRENT_LIMIT_OPERATIONAL_LIMIT_ACCEPTABLE_DURATION = "acceptableDuration";
+
+    public static final String ASSESSED_ELEMENT_FILE_KEYWORD = "AE";
+
+    public static final String ENTSOE_NS_NC_URL = "http://entsoe.eu/ns/nc";
+
+    public static final List<String> CURRENT_LIMIT_POSSIBLE_ALIASES_BY_TYPE_LEFT = List.of("CGMES.Terminal1", "CGMES.Terminal_Boundary_1");
+
+    public static final List<String> CURRENT_LIMIT_POSSIBLE_ALIASES_BY_TYPE_RIGHT = List.of("CGMES.Terminal2", "CGMES.Terminal_Boundary_2");
+
+    public static final String IEC_URL = "http://iec.ch/TC57/";
+
+    public enum ElementCombinationConstraintKind {
+        CONSIDERED(ENTSOE_NS_NC_URL + "#ElementCombinationConstraintKind.considered"),
+        INCLUDED(ENTSOE_NS_NC_URL + "#ElementCombinationConstraintKind.included"),
+        EXCLUDED(ENTSOE_NS_NC_URL + "#ElementCombinationConstraintKind.excluded");
+
+        ElementCombinationConstraintKind(String name) {
+            this.name = name;
+        }
+
+        private final String name;
+
+        public String toString() {
+            return this.name;
+        }
+    }
```

Comment:
I was thinking of a lighter way to declare the constants and to include the urls in the toString method

```java
public enum ElementCombinationConstraintKind {
    CONSIDERED("considered"),
    INCLUDED("included"),
    EXCLUDED("excluded");

    ElementCombinationConstraintKind(String name) {
        this.name = name;
    }

    private final String name;
    private final String url = ENTSOE_NS_NC_URL + "#ElementCombinationConstraintKind.";

    public String toString() {
        return this.url + this.name;
    }
}
```